hlx_statics/scripts/scripts.js

```diff
@@ -163,11 +143,186 @@ async function loadEager(doc) {
+          const environment = 'stg1';
+          const logsEnabled = true;
+        
+          setIMSParams(client_id, scope, environment, logsEnabled, resolve, reject, timeout); 
+        } else {
+          const client_id = 'stage_adobe_io';
+          const scope = 'AdobeID,openid,unified_dev_portal,read_organizations,additional_info.projectedProductContext,additional_info.roles,gnav,read_pc.dma_bullseye,creative_sdk';
+          const environment = 'stg1';
+          const logsEnabled = true;
+        
+          setIMSParams(client_id, scope, environment, logsEnabled, resolve, reject, timeout); 
+        }
+        
+        window.marketingtech = {
+          adobe: {
+            launch: {
+              property: 'global',
+              environment: 'dev',
+            },
+            analytics: {
+              additionalAccounts: 'pgeo1xxpnwadobeio-qa',
+            },
+          },
+        };
+      } else if (!isHlxPath(window.location.host) && !isStageEnvironment(window.location.host)) {
+        if (window.location.pathname.includes('/photoshop/api')) {
+          const client_id = 'cis_easybake';
+          const scope = 'AdobeID,openid,creative_sdk,creative_cloud,unified_dev_portal,read_organizations,additional_info.projectedProductContext,additional_info.roles,gnav,read_pc.dma_bullseye';
+          const environment = 'prod';
+          const logsEnabled = false;
+        
+          setIMSParams(client_id, scope, environment, logsEnabled, resolve, reject, timeout); 
+        } else {
+          const client_id = 'adobe_io';
+          const scope = 'AdobeID,openid,unified_dev_portal,read_organizations,additional_info.projectedProductContext,additional_info.roles,gnav,read_pc.dma_bullseye,creative_sdk';
+          const environment = 'prod';
+          const logsEnabled = false;
+        
+          setIMSParams(client_id, scope, environment, logsEnabled, resolve, reject, timeout); 
+        }
+
+        window.marketingtech = {
+          adobe: {
+            launch: {
+              property: 'global',
+              environment: 'production',
+            },
+            analytics: {
+              additionalAccounts: 'pgeo1xxpnwadobeio-prod',
+            },
+          },
+        };
+      }
+  
+      if (isHlxPath(window.location.host) || isStageEnvironment(window.location.host)) {
+        addExtraScript(document.body, 'https://auth-stg1.services.adobe.com/imslib/imslib.js');
+      } else {
+        addExtraScript(document.body, 'https://auth.services.adobe.com/imslib/imslib.min.js');
+      }
+    });
+  return window.imsLoaded;
+}
+
+if (window.adobeImsFactory && window.adobeImsFactory.createIMSLib) {
+  window.adobeImsFactory.createIMSLib(window.adobeid);
+}
+
+if (window.adobeIMS && window.adobeIMS.initialize) {
+  window.adobeIMS.initialize();
+}
+
```

Comment:
This seems to be dependant on IMS so we should do it post IMS loading. Maybe move this into loadLazy right after await loadIms() is resolved.